.github/workflows/test.yml

```diff
@@ -30,6 +30,7 @@ jobs:
       REDIS_PORT: 6379
       REDIS_PASSWORD: redis
       REDIS_STANDALONE_MODE: "true"
+      QUEUE_PREFIX: "{sims-api-test}"
```

Comment:
I was under the impression that the prefix would be just `DEV/TEST/QA/PROD`. Is it just for testing? If it is I would say `TEST` would be enough, IMO.